### ISSUE TITLE
panic: runtime error: index out of range [0] with length 0

### ISSUE THREAD

**Issue by k1LoW** [Songmu/tagpr#130]
Hi @Songmu.

I got the following [error with v1.0.5](https://github.com/k1LoW/grpcstub/actions/runs/3214282163/jobs/5254562486#step:4:118). [v1.0.4 works fine](https://github.com/k1LoW/grpcstub/actions/runs/3214308991/jobs/5254609603
).

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/Songmu/tagpr.(*tagpr).Run(0xc0001724d0, {0xa04e70, 0xc0000aa000})
	github.com/Songmu/tagpr/tagpr.go:326 +0x3026
github.com/Songmu/tagpr.Run({0xa04e70, 0xc0000aa000}, {0xc0000a01f0, 0x0, 0x0}, {0xa02600?, 0xc0000b0008?}, {0xa02600?, 0xc0000b0010})
	github.com/Songmu/tagpr/cli.go:37 +0x336
main.main()
	github.com/Songmu/tagpr/cmd/tagpr/main.go:14 +0x89
```

Thank you for a great tool. I was also able to avoid this issue because I had the option to change the version.

**Comment by Songmu** [comment]
Oops. Thank you for reporting.

I fixed at https://github.com/Songmu/tagpr/pull/131 and released it as v1.0.6. Please try it.

**Comment by k1LoW** [comment]
Thanks for the quick fix!

I just confirmed it [works fine with v1.0.6](https://github.com/k1LoW/grpcstub/actions/runs/3214417533/jobs/5254795031#step:4:3).
